openupgrade_scripts/scripts/stock/15.0.1.1/pre-migration.py

```diff
@@ -0,0 +1,42 @@
+from openupgradelib import openupgrade
+
+
+def _create_column_for_avoiding_automatic_computing(env):
+    openupgrade.logged_query(
+        env.cr,
+        """
+            ALTER TABLE stock_move
+                ADD COLUMN IF NOT EXISTS reservation_date date;
+        """,
+    )
+
+
+def _fill_stock_quant_package_name_if_null(env):
+    openupgrade.logged_query(
+        env.cr,
+        """
+            UPDATE stock_quant_package
+            SET name = 'Unknown Pack'
```

Comment:
Is there a chance we get a lot of these? 

Maybe 
```suggestion
            SET name = 'Unknown Pack ' || id::text
```
I never use packages so maybe it is irrelevant, just a comment.

Reply:
This field is required from 15.0, but at 14.0 it was set default value via `self.env['ir.sequence'].next_by_code('stock.quant.package')` and didn't insert by SQL before. So I think we don't need set value for it during migration from 14.0 to 15.0.

Reply:
After reviewing, I think we still need to migrate here. Although `stock.quant.package` are usually created via transfers but it has some views and end user can edit it. It mean `name` of `stock.quant.package` can be blank.